src/models/embedders/ttr.py

```diff
@@ -0,0 +1,10 @@
+from torch import nn
+
+
+class VanillaEmbedder(nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, batch):
+        output = {"embeddings": [batch["input_ids"], batch["attention_mask"], batch["token_type_ids"]]}
```

Comment:
No, this is not what we want. You should apply `nn.Embeddings` layer to `batch["input_ids"]` here and return `[batch size; seq len; emb dim]`